devel/ci/integration/bodhi/Dockerfile

```diff
@@ -0,0 +1,125 @@
+FROM registry.fedoraproject.org/fedora:29
+MAINTAINER "Aurelien Bompard" <abompard@fedoraproject.org>
+LABEL \
+  name="bodhi-web" \
+  vendor="Fedora Infrastructure" \
+  license="MIT"
+
+RUN curl -o /etc/yum.repos.d/infra-tags.repo https://infrastructure.fedoraproject.org/cgit/ansible.git/plain/files/common/fedora-infra-tags.repo
+
+# While dnf has a --nodocs, it doesen't have a --docs...
+RUN sed -i '/nodocs/d' /etc/dnf/dnf.conf
+
+
+# The echo works around https://bugzilla.redhat.com/show_bug.cgi?id=1483553 and any other future dnf
+# upgrade bugs.
+#RUN dnf upgrade -y || echo "We are not trying to test dnf upgrade, so ignoring dnf failure."
+
+# Install Bodhi deps
+# 1. bodhi-server
+# 2. bodhi-composer
+# 3. python3-bodhi-client
+RUN dnf install -y \
+    \
+    httpd \
+    intltool \
+    liberation-mono-fonts \
+    python3-dogpile-cache \
+    python3-koji \
+    python3-librepo \
+    python3-mod_wsgi \
+    python3-pillow \
+    "python3dist(alembic)" \
+    "python3dist(arrow)" \
+    "python3dist(bleach)" \
+    "python3dist(click)" \
+    "python3dist(colander)" \
+    "python3dist(cornice)" \
+    "python3dist(cryptography)" \
+    "python3dist(fedmsg)" \
+    "python3dist(feedgen)" \
+    "python3dist(jinja2)" \
+    "python3dist(kitchen)" \
+    "python3dist(markdown)" \
+    "python3dist(psycopg2)" \
+    "python3dist(pylibravatar)" \
+    "python3dist(pyramid)" \
+    "python3dist(pyramid-fas-openid)" \
+    "python3dist(pyramid-mako)" \
+    "python3dist(python-bugzilla)" \
+    "python3dist(python-fedora)" \
+    "python3dist(python3-openid)" \
+    "python3dist(simplemediawiki)" \
+    "python3dist(six)" \
+    "python3dist(sqlalchemy)" \
+    "python3dist(waitress)" \
+    \
+    python3-createrepo_c \
+    python3-hawkey \
+    skopeo \
+    \
+    /usr/bin/koji \
+    python3-dnf \
+    python3-koji \
+    "python3dist(click)" \
+    "python3dist(iniparse)" \
+    "python3dist(kitchen)" \
+    "python3dist(python-fedora)" \
+    "python3dist(six)" \
+    \
+    fedmsg \
+    fedmsg-base \
```

Comment:
A lot of the above is in common with the existing Fedora 29 unit test Dockerfile. I wonder if we should make a shared container layer that this Dockerfile and the other one use. Then ```bodhi-ci``` can build and tag that layer first, and then this and the other one can ```FROM``` that intermediate layer?